config/samples/gateway_v1_envoyfleet.yaml

```diff
@@ -3,7 +3,7 @@ kind: EnvoyFleet
 metadata:
   name: default
 spec:
-  image: "envoyproxy/envoy-alpine:v1.20.0"
+  image: "docker.io/envoyproxy/envoy:v1.23.0"
```

Comment:
Comparing these two images I see that alpine image is at 28MB where plain envoy is 50MB. But the only drawback of the alpine images is that it doesn't seem to be supported anymore.

Reply:
That's correct. The Alpine image does not receive updates any more.